python/delta_sharing/reader.py

```diff
@@ -172,11 +174,19 @@ def __to_pandas_kernel(self):
         return result
 
     def to_pandas(self) -> pd.DataFrame:
-        response_format = self._rest_client.autoresolve_query_format(self._table)
+        response_format = ""
+        # If client indicates to use delta format use it, otherwise autoresolve the format
+        if (self._useKernel is not None):
```

Comment:
nit: does this work? `if (self._user_delta_format is not None and self._use_delta_format):`

Reply:
yeah None is false in Pyhton so i can actually just do: `if self._useKernel`